vtex/reliability-research
# Input_maker to use only round-robin for request balancing

#### What is the purpose of this pull request?

This PR is part of the work done in #97.
Aims to change the `input_maker` to no longer direct all requests to a load balancer but to a list of addresses in a round-robin fashion.

#### What problem is this solving?

Currently we need two parsers to parse the canonical load to a replayer input in round-robin style. After this PR, executing the `input_maker` will be enough.

#### How should this be manually tested?
Using our Drive's `data_xml_solr.txt`, `queries.csv` and `downsample/search.csv`, run the input_maker.py with the following command:
```
python3 input_maker.py --downsample_file=./downsample/search.csv --queries_file=queries.csv --body_file=data_xml_solr.txt --output_file=input_robin_test --number_of_docs=5000 --round_robin_urls="http://172.31.93.251:8983, http://172.31.81.203:8983, http://172.31.41.199:8983, http://172.31.46.159:8983"
```
After this, check if `input_robin_test` have the correct addresses

#### Screenshots or example usage

#### Types of changes
- New feature (non-breaking change which adds functionality)
